packages/services/src/session/restapi.ts

```diff
@@ -100,16 +100,37 @@ export async function startSession(
   settings: ServerConnection.ISettings = ServerConnection.makeSettings()
 ): Promise<Session.IModel> {
   const url = URLExt.join(settings.baseUrl, SESSION_SERVICE_URL);
-  const init = {
+  const body = JSON.stringify(options)
+  const bodyjson = JSON.parse(body);
```

Comment:
you should online this to better show the intent and call it something like optionsCopy